about/index.html

```diff
@@ -203,7 +203,7 @@ <h5 class="subheader">The members of the OME consortium are as follows:</h5>
         <hr>
         <div class="row medium-up-1 large-up-1 text-center consortium-list">
             {% for member in page.consortium %}
-            <div class="large-expand columns"><a target="_blank" href="{{ member.member_link }}">{{ member.member_name }}</a> <span class="institution">{{ member.institution }}</span></div>
+            <div class="large-expand columns"><a target="_blank" href="{{ member.member_link | replace: '!SITE_BASEURL!', site.baseurl }}">{{ member.member_name }}</a> <span class="institution">{{ member.institution }}</span></div>
```

Comment:
This and the same below is to ensure that a breaking change would be flagged during a PR review?